### ISSUE TITLE
<input type='checkbox' checked={false} /> not rendered properly.

### ISSUE THREAD

**Issue by dtinth** [developit/preact-render-to-string#2]
## Expected result

```html
<input type="checkbox">
```

## Actual result

```html
<input type="checkbox" checked="false"></input>
```

Note that this HTML code generates a checkbox that is checked, because the presence of the `checked` attribute.

**Comment by developit** [comment]
Good catch, I'll get a fix in for this right away. Thanks!

**Comment by developit** [comment]
Fixed and release in [2.1.0](https://github.com/developit/preact-render-to-string/releases/tag/2.1.0).
Thanks again!
